website/source/docs/builders/googlecompute.html.md

```diff
@@ -125,8 +125,11 @@ builder.
 -   `project_id` (string) - The project ID that will be used to launch instances
     and store images.
 
--   `source_image` (string) - The source image to use to create the new
-    image from. Example: `"debian-7-wheezy-v20150127"`
+-   `source_image` or `source_image_family` (string) - The source image or
+    family to use to create the new image from. The image family always returns
+    its latest image that is not deprecated. If both `source_image` and
+    `source_image_family` are specified, `source_image` takes precedence.
+    Example: `"debian-7-wheezy-v20150127"`, `"debian-7"`.
```

Comment:
Keep both under required and split them. Required is more like _doesn't have a default value and might be required_, see https://www.packer.io/docs/builders/virtualbox-iso.html#iso_checksum